bitnami/keycloak/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/bitnami-docker-keycloak
   - https://github.com/keycloak/keycloak
-version: 4.2.0
+version: 4.2.1
```

Comment:
I would bump the minor version here instead according to [semver](https://semver.org/)

Reply:
Yes you're right, I fixed that :)